tasks/main.yml

```diff
@@ -5,36 +5,8 @@
 - include: pkg-redhat.yml
   when: ansible_os_family == "RedHat"
 
-- name: Create main Datadog agent configuration file
-  template:
-    src: datadog.conf.j2
-    dest: /etc/dd-agent/datadog.conf
-    owner: '{{ datadog_user }}'
-    group: '{{ datadog_group }}'
-  notify: restart datadog-agent
+- include: agent5.yml
+  when: datadog_agent6 == false
```

Comment:
are you sure the comparison works as expected? (want to make sure it's actually a boolean comparison and not a string comparison: for instance, if we set `datadog_agent6: no` in the variables, does this still evaluate to the `True` boolean?)

Reply:
It does work, but you're right it not the best. I move condition to explicit Boolean check.